### PR TITLE
Install dependencies and format hfvqe/quickstart

### PR DESCRIPTION
**Pull request by mpharrigan**
Add notebook buttons, installation step, and some very minor code style adjustments. 

Partially address #101. Re-work `molecular_exmaple.py` to
support putting these data files elsewhere in the future. Run `nbfmt`.

cc #83

### CODE REVIEW

File: docs/hfvqe/quickstart.ipynb [hunk 1]
```diff
@@ -64,7 +129,8 @@
     "                           constant=molecule.nuclear_repulsion,\n",
```
Comment (lamberta) [inline docs/hfvqe/quickstart.ipynb:129]: Error in Colab, expected?

`rhf_objective, molecule, parameters, obi, tbi = make_h6_1_3()` ->

```
OSError                                   Traceback (most recent call last)
<ipython-input-4-c2912b0a45e7> in <module>()
----> 1 rhf_objective, molecule, parameters, obi, tbi = make_h6_1_3()
      2 ansatz, energy, gradient = rhf_func_generator(rhf_objective)
      3 
      4 # settings for quantum resources
      5 qubits = [cirq.GridQubit(0, x) for x in range(molecule.n_orbitals)]

4 frames
/usr/local/lib/python3.6/dist-packages/h5py/_hl/files.py in make_fid(name, mode, userblock_size, fapl, fcpl, swmr)
    171         if swmr and swmr_support:
    172             flags |= h5f.ACC_SWMR_READ
--> 173         fid = h5f.open(name, flags, fapl=fapl)
    174     elif mode == 'r+':
    175         fid = h5f.open(name, h5f.ACC_RDWR, fapl=fapl)

h5py/_objects.pyx in h5py._objects.with_phil.wrapper()

h5py/_objects.pyx in h5py._objects.with_phil.wrapper()

h5py/h5f.pyx in h5py.h5f.open()

OSError: Unable to open file (unable to open file: name = '/usr/local/lib/python3.6/dist-packages/recirq/hfvqe/molecular_data/hydrogen_chains/h_6_sto-3g/bond_distance_1.3/H6_sto-3g_singlet_linear_r-1.3.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
```

Comment (mpharrigan) [reply docs/hfvqe/quickstart.ipynb:129]: sorry; I should start writing these up front: this error is fixed by the change to `setup.py` in this PR, and therefore won't work until this is merged. I tested locally in a fresh venv to 1) reproduce the error in #101 and 2) verify the bandaid fix

**Review by lamberta — CHANGES_REQUESTED** [review]
While you're in here, can you shorten the heading title for: `### Generate the input files, set up quantum resources, and set up the OpdmFunctional to make measurements. ` ?

Can use a shorter section title so it doesn't run over in the nav and use that as the setence description that follows.

**Comment by mpharrigan** [comment]
reworked the headings

**Review by lamberta — APPROVED** [review]
Nice, thanks

**Comment by mpharrigan** [comment]
I'll give @ncrubin a beat to look this over, just in case

**Pull request merged**